.github/workflows/scripts/osm_deploy.sh

```diff
@@ -16,10 +16,10 @@ fi
 
 curl -fsL https://raw.githubusercontent.com/openservicemesh/osm-docs/main/manifests/apps/bookstore.yaml --output bookstore.yaml
 sleep 10
-mesheryctl system login --provider None
-mesheryctl mesh deploy adapter meshery-osm:10009
+#mesheryctl system login --provider None
+echo | mesheryctl mesh deploy adapter meshery-osm:10009 --token "./.github/workflows/auth.json"
 echo "Onboarding application... Standby for few minutes..."
-mesheryctl app onboard -f "./bookstore.yaml"
+mesheryctl app onboard -f "./bookstore.yaml" -s "Kubernetes Manifest" -t "./.github/workflows/auth.json"
```

Comment:
Same. Let's just use `--token` here just for consistency

Reply:
```suggestion
mesheryctl app onboard -f "./bookstore.yaml" -s "Kubernetes Manifest" --token "./.github/workflows/auth.json"
```